### ISSUE TITLE
concourse-lite 3.9.0 fails on Debian 9

### ISSUE THREAD

**Issue by walter-db** [concourse/concourse#2046]
Bug Report

I am heaving issues with starting up with Concourse.
As Instructed on https://concourse.ci/concourse-lite.html installed bosh CLI, downloaded the latest Concourse Lite from https://github.com/concourse/concourse/releases/download/v3.9.0/concourse-lite.yml but when starting up using bosh create-env councouse-lite.yml I get the following error:

```
bosh create-env concourse-lite.yml
Deployment manifest: '/home/walter/concourse-lite-3.9.0/concourse-lite.yml'
Deployment state: '/home/walter/concourse-lite-3.9.0/concourse-lite-state.json'

Started validating
  Downloading release 'concourse'... Failed (00:00:03)
Failed validating (00:00:03)

Failed to download from 'https://github.com/concourse/concourse/releases/download/v3.9.0/concourse-3.9.0-3541.2.tgz':
  Verifying digest for downloaded file:
    Expected stream to have digest '6cc893c34ee4a504d59a06b2df3413d6e5a68e10' but was '43c8ef28fb100441efd0fb80244f979c7859f983'

Exit code 1
```

Ran the command from an empty directory (only the yaml file is in there)

- Concourse version: latest (3.9.0)
- Deployment type: BOSH (version 2.0.48-e94aeeb-2018-01-09T23:08:07Z)
Installing on Debian Stretch.

**Comment by walterngti** [comment]
Looks like after release creation, the yaml file has been updated. The hash is correct for the concourse version in the release, 3.9.0-3468-22.. This is however not the file mentioned in the yml file.

**Comment by taybin** [comment]
This fails on OSX too, fwiw.

**Comment by mjmeintjes** [comment]
The same error on ubuntu 17.04

**Comment by alvaropinot** [comment]
The hash seems to be miss aligned between the 3.9.0 `concourse-lite.yml` file pointing to `6cc893c34ee4a504d59a06b2df3413d6e5a68e10` and the actual tgz file from https://github.com/concourse/concourse/releases/download/v3.9.0/concourse-3.9.0-3541.2.tgz which is `43c8ef28fb100441efd0fb80244f979c7859f983`.

In case it helps you might download the previous release (3.8.0) while the Concourse team releases a new yml file by running:

```sh
wget https://github.com/concourse/concourse/releases/download/v3.8.0/concourse-lite.yml
```

and then just create the env as instructed before by running:

```sh
bosh create-env concourse-lite.yml
```

**Comment by leeDav** [comment]
I'm also experiencing the issue, with both 3.9.0 and the newly released 3.9.1.

3.8.0 works fine, though.

**Comment by xtremerui** [comment]
closing stale issue.
